webext/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "cardano-wallet",
+  "type": "module",
+  "version": "0.0.0",
+  "description": "A dev cardano wallet that works in all browsers",
+  "author": "MLabs <contact@mlabs.city>",
+  "license": "MIT",
+  "private": true,
+  "scripts": {},
+  "devDependencies": {
+    "crx": "^5.0.1",
```

Comment:
```suggestion
    "crx": "^5.0.1",
    "esbuild-plugins-node-modules-polyfill": "^1.6.1",
```

Reply:
Good find @nazrhom. Thanks!. I had accidentally removed it and didn't notice it was gone as it was still present in my `node_modules` folder.